.github/CODEOWNERS 2

```diff
@@ -0,0 +1,77 @@
+# Core
```

Comment:
Were these files under `.github/` added accidentally?

Reply:
Yes, apologies. Happy to delete them and make new commit if that would resolve the issue.